openmc/cmfd.py

```diff
@@ -7,9 +7,53 @@
 
 
 class CMFDMesh(object):
+    """A structured Cartesian mesh used for coarse mesh finite difference
```

Comment:
Also, it might be worth including a reference to CMFD (I've seen references included in some technical docstrings before). I think K. Smith's 1983 and/or 2002 papers might be good, though @samuelshaner's [docs](https://mit-crpg.github.io/OpenMOC/methods/cmfd.html#smith-2002) on CMFD would be a good place to pick up the most relevant references.